### PR TITLE
Add galaxy-related options for Ansible provisioner

### PR DESCRIPTION
**Pull request by dkasyanov**
### Issue
- ansible provisioner missing galaxy fields that's available in Packer
- ansible-local has misspelled `galaxy_command` option

### List of Changes Proposed
Ansible Provisioner:
- galaxy_file
- galaxy_command
- galaxy_force_install
- roles_path
- collections_path
Upstream documentation: https://www.packer.io/docs/provisioners/ansible#galaxy_file

Ansible Local Provisioner:
- galaxy_command
Upstream documentation: https://www.packer.io/docs/provisioners/ansible-local#galaxy_command

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,18 +1,9 @@
...
-.. image:: https://travis-ci.org/mayn/packerlicious.svg?branch=master
-    :target: https://travis-ci.org/mayn/packerlicious
-
-.. image:: https://ci.appveyor.com/api/projects/status/9av1rr1li2ah25gs/branch/master?svg=true
-    :target: https://ci.appveyor.com/project/mayn/packerlicious
-
-.. image:: https://coveralls.io/repos/github/mayn/packerlicious/badge.svg?branch=master
-    :target: https://coveralls.io/github/mayn/packerlicious
 
+.. image:: https://ci.appveyor.com/api/projects/status/734n8y5s0wtqspj7/branch/master?svg=true
```
Comment (mayn) [inline README.rst]: what are the badge edits for?

File: README.rst [hunk 2]
```diff
@@ -194,7 +185,7 @@ packerlicious contains source code from `troposphere`_ which is licensed under t
 
 .. _`packer`: https://www.packer.io/
 .. _`troposphere`: https://github.com/cloudtools/troposphere
-.. _`LICENSE`: https://github.com/mayn/packerlicious/blob/master/LICENSE
+.. _`LICENSE`: https://github.com/dkasyanov/packerlicious/blob/master/LICENSE
```
Comment (mayn) [inline README.rst]: why change in license?

**Review by mayn — CHANGES_REQUESTED** [review]

File: README.rst (hunk 2 above)
Comment (dkasyanov) [reply README.rst]: sorry. I  was building project for myself and forgot about this PR.
reverted changes

File: README.rst (hunk 1 above)
Comment (dkasyanov) [reply README.rst]: reverted

**Pull request closed without merging**